packages/analytics/analytics-chart/src/components/chart-types/DoughnutChart.vue

```diff
@@ -191,21 +238,78 @@ const chartInstance = ref<Chart>()
  * When in Preview mode, Chart and Legend are vertically stacked, and the
  * Legend list items are allowed to spread horizontally.
  */
-const chartFlexClass = (position: `${ChartLegendPosition}`) => {
+const chartFlexClass = computed<string>(() => {
   return {
     [ChartLegendPosition.Right]: 'legend-row',
     [ChartLegendPosition.Bottom]: 'column',
     [ChartLegendPosition.Hidden]: 'hidden',
-  }[position]
-}
+  }[props.legendPosition]
+})
 
 const tooltipDimensions = ({ width, height }: { width: number, height: number }) => {
   tooltipData.width = width
   tooltipData.height = height
 }
+
+// When displaying a simple chart, we only expect two values in the dataset
+const metricHighlight = computed(() => approxNum(formattedDataset?.value[0]?.data[0], { capital: true }))
+const metricTotal = computed(() => approxNum(formattedDataset?.value[0]?.data[0] + formattedDataset?.value[0]?.data[1], { capital: true }))
+
+const metricHighlightColor = computed(() => `color: ${formattedDataset?.value[0]?.backgroundColor[0]}`)
+
+// Conditionally show large or small metric value, or neither
+const showMetricLarge = computed(() => [ChartMetricDisplay.Full, ChartMetricDisplay.SingleMetric].includes(metricDisplayRef.value))
+const showMetricSmall = computed(() => metricDisplayRef.value === ChartMetricDisplay.Full)
```

Comment:
Rather than doing this in here, I'd just make a separate, simpler, GuageChart internal component that only does the stuff pertaining to a "simple chart"

Reply:
Same response as above. There's more in common than not in common...